subsys/net/ip/tcp2.c

```diff
@@ -376,10 +376,21 @@ static const char *tcp_conn_state(struct tcp *conn, struct net_pkt *pkt)
 	return buf;
 }
 
-static bool tcp_options_check(void *buf, ssize_t len)
```

Comment:
in your commit message: "this does not work if ..."

Reply:
> in your commit message: "this does not work if ..."

Thanks, will fix